### DISCUSSION TITLE
cannot sign in

### DISCUSSION THREAD

**Discussion by ZhouDivd** [community/community#14325] [Copilot Conversations]
I use vs2022  ,everytime I login ,A device activation code I submit on web.but it can not work. shows "NotSignedIn".And it works in vs
code

**Comment by Daniel3356** [comment]
When have you request to use Github copilot? I make a request on January and I recently got approved.  There is a waiting list I thinks that is why isn't working for you.

**Reply by ZhouDivd** [reply]
Thanks a lot！I tried many times.Now Copilot can work.
